core/src/main/scala/filodb.core/query/RangeVector.scala

```diff
@@ -84,6 +84,10 @@ object CustomRangeVectorKey {
   val emptyAsZcUtf8 = toZcUtf8(empty)
 }
 
+
+object RangeVector {
```

Comment:
I think I like Java static field over companion objects.

Reply:
Hmm... I wouldn't have any issues with moving this code to Java when everything is proven and tested.